openwisp_controller/config/templates/admin/config/change_form.html

```diff
@@ -61,29 +61,69 @@
         // enable default templates - do not remove this comment
         (function ($) {
             var urls = {{ default_template_urls| safe }},
-                firstRun = true;
-            $('#id_organization').change(function () {
-                var url = urls[$(this).val()],
-                    isNew = $('#id_config-0-id').length == 0;
-                // if device is not new, do not execute on page load
-                if (!isNew && firstRun) {
-                    return
-                }
-                firstRun = false;
-                // if no organization
-                if (!url) {
+                firstRun = true,
+                backendField = $('#id_config-0-backend'),
```

Comment:
The JS script code has become quite long and it's not good to keep it as an inline script.
Can you please move this into its own JS file, loaded only in the Device admin?

The result could be a function, eg:

```javascript
bindDefaultTemplateLoading({{ default_template_urls| safe }});
```

This way the script gets cached, quality checked and is also consistent with the rest of the JS code.